languages.toml

```diff
@@ -3840,11 +3840,14 @@ source = { git = "https://github.com/Decurity/tree-sitter-circom", rev = "021505
 name = "snakemake"
 scope = "source.snakemake"
 roots = ["Snakefile", "config.yaml", "environment.yaml", "workflow/"]
-file-types = ["smk", "Snakefile"]
+file-types = ["smk", { glob = "Snakefile" } ]
```

Comment:
In the original commit, "Sankefile" was handled as a suffix, when instead Helix should look at the full filename to apply the language.  Like `Make` and `Makefile`, the `Snakemake` pipeline is defined in `Snakefile`. I added the required `glob` part.